templates/windows10.tpl.yaml

```diff
@@ -144,13 +144,20 @@ objects:
               runtime: {}
               reset: {}
           devices:
+{% if item.multiqueue %}
+            networkInterfaceMultiqueue: True
+{% endif %}
             disks:
             - disk:
                 bus: sata
               name: ${NAME}
             interfaces:
             - masquerade: {}
+{% if item.multiqueue %}
+              model: virtio
```

Comment:
Btw. don't need virtio drivers to be installed in windows image?

Reply:
Ah, I thought they were already in these images we use. I will keep an eye on it, especially for the Windows 10 test.

Regarding users, the idea is to require them to have it set up properly before using the high performance templates.